lib/ContainerManager.js

```diff
@@ -392,15 +408,26 @@ Server.prototype.routes.get['containers'] = function(req, res, next) {
       infos = infos.map(function(info){
         return {
           id: info.Id,
-          name: info.Name,
+          name: info.Name.substr(1), // get rid of initial '/' character
           state: info.State,
           ports: info.NetworkSettings.Ports
         }
       })
 
+      function stateReducer(running){
+        return function(count, info){
+          var state = info.state || info.State
```

Comment:
Adding case switching code to save on not having to type out `State: state` seems silly, we're adding a code path (that, ideally we'll have a test for) to use the fancy new syntax instead of typing even fewer characters where we instantiate this object.